packages/presentation/lib/screens/home/home_bloc.dart

```diff
@@ -37,11 +41,13 @@ class _HomeBloc extends BlocImpl<BaseArguments, HomeData> implements HomeBloc {
   final GetNowShowingMoviesUseCase _getNowShowingMoviesUseCase;
   final GetComingSoonMoviesUseCase _getComingSoonMoviesUseCase;
   final GetImageUrlUseCase _getImageUrlUseCase;
+  final LogAnalyticsEventUseCase _logAnalyticsEventUseCase;
```

Comment:
move to BlocImpl as you use it everywhere and no need to inject it each time

Reply:
Fixed.